specs/LocaleRegion.md

```diff
@@ -0,0 +1,111 @@
+LocaleRegion
+===
+
+# Background
+Developers have requested access to programmatically choose the specific locale/region to use in WebView2. 
+The locale/region updates the JavaScript region (objects in the ECMA standard) which gets reflected as
+variations in time and date formats. Currently, the locale is by default set to the same value as the 
+display language. 
+
+You can use the `LocaleRegion` API to update the locale value individually from the display
+language.
+
+# Description
+* The intended value for `LocaleRegion` is in the format of `language[-country]` where `language` is the 
+2-letter code from [ISO 639](https://www.iso.org/iso-639-language-codes.html) and `country` is the 
+2-letter code from [ISO 3166](https://www.iso.org/standard/72482.html).

```

Comment:
Maybe just say "in the same format as the CoreWebView2EnvironmentOptions.Language property" (assuming that's true). That way, there's only one place you have to document language tags.

Reply:
Yes. Assuming these are the same (I believe so but please verify) please have this reference the Language property like Raymond says. Please also validate the Language property currently has a good description of its input in its reference documentation and if not open a bug for that.

Reply:
Yes but is that correct or was it also a mistake?